tests/rptest/tests/wasm_failure_recovery_test.py

```diff
@@ -81,6 +81,7 @@ def verify_results(self):
         return materialized_result_set_compare
 
 
+"""
 class WasmMeshFailureRecoveryTest(WasmFailureRecoveryTest):
```

Comment:
Please could you use the `@ignore` decorator, with a comment linking a github issue for fixing the test -- it helps to track which tests are disabled and why.

Reply:
Great comment, I actually had no idea about that decorator, +1

Reply:
Ah, I see what you mean.  These tests could perhaps be reorganized a bit to make all the tests enter through unique test methods rather than the same verify_materialized_topics_test for all, but it's not necessary right now.

Reply:
cc: @graphcareful